extensions/interactions/InteractiveMap/directives/interactive-map-validation.service.spec.ts

```diff
@@ -72,22 +73,24 @@ describe('InteractiveMapValidationService', () => {
         value: 0
       }
     };
-    goodAnswerGroups = [agof.createNew(
-      [rof.createFromBackendDict({
-        rule_type: 'Within',
-        inputs: {
-          d: 100
-        }
-      }), rof.createFromBackendDict({
-        rule_type: 'NotWithin',
-        inputs: {
-          d: 50
-        }
-      })],
+    const goodAnswerGroup = agof.createNew(
```

Comment:
I am starting to wonder ... should createNew() never allow a rule inputs arg at all? If you don't want people to use it, then that might be the logical way to proceed. (If so, though, you should add a note to explain that rules should be added after the answer group is created.)